### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request make use of webpack+npm instead of webassets+bower for theme handling.

This is currently a work in progress

Connects to etalab/udata#74

### CODE REVIEW

File: theme/less/gouvfr/mixins.less [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /**
  * Common site styles and helpers.
  */
-@import "udata/mixins.less";
+// @import "udata/mixins.less";
```
Comment (davidbgk) [inline theme/less/gouvfr/mixins.less]: To remove.

File: theme/less/gouvfr/navbars.less [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
-    background-image: url('../img/datawatch-bulles.png');
+    background-image: url('/img/datawatch-bulles.png');
     background-size: cover;
     position: absolute;
     min-height: 100%;
     min-width: 100%;
 }
 
-@topic-png:  '../img/arcs.png';
+// @topic-png:  '/img/arcs.png';
```
Comment (davidbgk) [inline theme/less/gouvfr/navbars.less]: To remove.

File: theme/less/gouvfr-dataset.less [hunk 3]
```diff
@@ -0,0 +1,364 @@
+// @import "gouvfr.less";
```
Comment (davidbgk) [inline theme/less/gouvfr-dataset.less]: Remove?

File: udata_gouvfr/theme/templates/dataset/display.html [hunk 4]
```diff
@@ -9,10 +9,16 @@
...
 
+{% set body_class = 'dataset-display' %}
+
+{% block extra_js %}
+<script type="text/javascript" src="{{ static('dataset/display.js') }}"></script>
+{% endblock %}
+
 {% block extra_css %}
-{% assets 'gouvfr-dataset-css' %}
+{#% assets 'gouvfr-dataset-css' %}
```
Comment (davidbgk) [inline udata_gouvfr/theme/templates/dataset/display.html]: Remove?

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    devtools: 'eval-source-map',
+    module: {
+        loaders: [
+            {test: /img\/.*\.(jpg|jpeg|png|gif|svg)$/, loader: 'file?name=[path][name].[ext]'},
+            {test: /\.css$/, loader: css_loader},
+            {test: /\.less$/, loader: less_loader},
+            {test: /\.json$/, loader: "json"},
+            {test: /\.html$/, loader: "html"},
+            {test: /\.(woff|svg|ttf|eot|otf)([\?]?.*)$/, loader: "file?name=[name].[ext]"},
```
Comment (davidbgk) [inline webpack.config.js]: `woff2` too?

**Comment by davidbgk** [comment]
r+

File: webpack.config.js (hunk 5 above)
Comment (noirbizarre) [inline webpack.config.js]: It's already taken in account by the regex

**Pull request merged**